Documentation/CodeSnippets/TypoScriptSyntax/OperatorUnset.rst.txt

```diff
@@ -0,0 +1,15 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/ObjectUnset/setup.typoscript
+
+   myIdentifier.mySubIdentifier = TEXT
+   myIdentifier.mySubIdentifier = myValue
+   myIdentifier.mySubIdentifier.stdWrap = <p>|</p>
+
+   # "myIdentifier.mySubIdentifier" is completely removed, including value
+   # assignment and sub identifier "stdWrap"
+   myIdentifier.mySubIdentifier >
+
+   # Same as above: Everything after ">" operator is considered a comment
+   myIdentifier.mySubIdentifier > // Some comment
```

Comment:
Not sure if it should be mentioned that for fluid templates it might be required to unset the values only but not the variables or keys.

Reply:
That is out of context here I would say

Reply:
where would be the right context for mentioning it?

Reply:
Let us bring this big PR in: It is a huge improvement. Subsequent changes can be made by additional PRs.